cloudsync/smartsync.py

```diff
@@ -257,6 +258,22 @@ def _ent_to_smartinfo(self, ent: Optional[SyncEntry], local_dir_info: Optional[D
                            )
         return retval
 
+    def _ignore_ent(self, ent: SyncEntry) -> bool:
+        '''Returns True if Smartsync should ignore the ent during listdir/info'''
+        if not ent:
+            return True
+
+        # Ent has been trashed locally or remotely
+        if ent[LOCAL].exists in (TRASHED, MISSING) \
+                    or ent[REMOTE].exists in (TRASHED, MISSING):
+            return True
+
+        # Local rename in process
+        if ent[LOCAL].path and self.translate(LOCAL, ent[REMOTE].path) != ent[LOCAL].path:
+            return True
```

Comment:
```suggestion
        if ent[LOCAL].path:
        	return self.translate(LOCAL, ent[REMOTE].path) == ent[LOCAL].path
```

Reply:
I think this is the opposite of what we want to do. We only want to ignore the SyncEnt if the paths don't match

Reply:
Right, this is reversed... I'd like to request that this be changed from `_ignore_ent` to `_include_ent`. I don't think that functions should ever be inherently falsy, it leads to confusion way too often

Reply:
Done